packages/-build-infra/bin/test-external-partner-project.js

```diff
@@ -8,12 +8,13 @@ const { shellSync } = require('execa');
 // apparently violates no-extraneous require? /shrug
 const debug = require('debug')('test-external');
 const rimraf = require('rimraf');
+const chalk = require('chalk');
 
-const projectRoot = path.resolve(__dirname, '../');
+const projectRoot = path.resolve(__dirname, '../../../');
```

Comment:
This is true even when run from project root?

Reply:
yes